dependencies/pip/dev_requirements.txt

```diff
@@ -5,13 +5,15 @@
 #    pip-compile --output-file dependencies/pip/dev_requirements.txt dependencies/pip/dev_requirements.in
 #
 -e git+https://github.com/dimagi/django-digest@0eb1c921329dd187c343b61acfbec4e98450136e#egg=django_digest
--e git+https://github.com/kobotoolbox/formpack.git@e7428a0af798c006b38fe54ac804840c96b1ecd0#egg=formpack
+-e git+https://github.com/kobotoolbox/formpack.git@99e485506d52eee6351a61354f64f0110f59cfd7#egg=formpack
 -e git+https://github.com/kobotoolbox/pyxform.git@2.017.36#egg=pyxform
 amqp==2.1.4
 anyjson==0.3.3
 begins==0.9
 billiard==3.5.0.2
+boto3==1.4.7
 boto==2.45.0
+botocore==1.7.33          # via boto3, s3transfer
 celery==4.0.2
 cffi==1.9.1               # via cryptography
 cryptography==1.7.2       # via paramiko, pyopenssl
```

Comment:
Can we make this 1.8 at some point? makes install go much easier

Reply:
Is that a dependency of `boto3` added here by `pip-compile`? If so, what version of `boto3` do we need to use to pull in `botocore` 1.8?

Reply:
@KeynesYouDigIt, I'll try `boto3` [1.5.8](https://github.com/boto/boto3/blob/1.5.8/setup.py#L17)